H/HelFEM/common.jl

```diff
@@ -64,9 +62,9 @@ products = [
 dependencies = [
     BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
     Dependency("libcxxwrap_julia_jll"),
-    Dependency(PackageSpec(name = "armadillo_jll", version = "9.850.1")),
-    Dependency(PackageSpec(name = "GSL_jll", version = "2.6.0")),
-    Dependency(PackageSpec(name = "OpenBLAS_jll", version = "0.3.9")),
+    Dependency("armadillo_jll", compat = "9.850.1"),
+    Dependency("GSL_jll", compat = "2.6.0"),
+    Dependency("OpenBLAS_jll", compat = "0.3.10"),
```

Comment:
Right, we two version numbers we can indicate: the version of the package we build with (second argument of `Dependency`, as `VersionNumber`), and the compat (`compat` keyword argument, as a string), in this case we need to specify them both because in principle they can be different):
```suggestion
    Dependency("OpenBLAS_jll", v"0.3.10"; compat = "0.3.10"),
```